src/applications/coronavirus-vaccination/components/Form.jsx

```diff
@@ -96,48 +101,58 @@ function Form({ formState, updateFormData, router, isLoggedIn, profile }) {
     [router, formState],
   );
 
-  if (!formState) {
-    // The form is being initialized into Redux. Wait til next render.
-    return null;
-  }
-
   if (submitStatus === requestStates.pending) {
     return <LoadingIndicator message="Submitting your form..." />;
   }
 
   return (
     <>
-      <h1>COVID-19 vaccines — Stay informed and help us prepare</h1>
+      <h1 id="covid-vaccination-heading-form" className="no-outline">
+        Fill out the form below to sign up
+      </h1>
+      <p>
+        We’ll send you regular updates on how we’re providing COVID-19 vaccines
+        across the country—and when you can get your vaccine if you want one.
+        You don't need to sign up to get a vaccine.
+      </p>
       {isLoggedIn ? (
         <p>
           <strong>Note:</strong> Any changes you make to your information here
           won’t change your information in your VA.gov profile or any other
           accounts.
         </p>
       ) : null}
-      <SchemaForm
-        addNameAttribute
-        // "name" and "title" are used only internally to SchemaForm
-        name="Coronavirus vaccination"
-        title="Coronavirus vaccination"
-        data={formState.formData}
-        schema={formState.formSchema}
-        uiSchema={formState.uiSchema}
-        onChange={onFormChange}
-        onSubmit={onFormSubmit}
-      >
-        {submitStatus === requestStates.failed ? (
-          <div className="vads-u-margin-bottom-2">
-            <AlertBox
-              status={ALERT_TYPE.ERROR}
-              content="An error occurred while trying to save your form. Please try again later."
-            />
-          </div>
-        ) : null}
-        <button type="submit" className="usa-button">
-          Submit
-        </button>
-      </SchemaForm>
+      {formState ? (
+        <SchemaForm
+          addNameAttribute
+          // "name" and "title" are used only internally to SchemaForm
+          name="Coronavirus vaccination"
+          title="Coronavirus vaccination"
+          data={formState.formData}
+          schema={formState.formSchema}
+          uiSchema={formState.uiSchema}
+          onChange={onFormChange}
+          onSubmit={onFormSubmit}
+        >
+          {submitStatus === requestStates.failed ? (
+            <div className="vads-u-margin-bottom-2">
+              <AlertBox
+                status={ALERT_TYPE.ERROR}
+                content="An error occurred while trying to save your form. Please try again later."
+              />
+            </div>
+          ) : null}
+          <button
+            type="submit"
+            className="usa-button"
+            aria-label="Sign up to stay informed about COVID-19 vaccines"
+          >
+            Sign up to stay informed
+          </button>
+        </SchemaForm>
+      ) : (
+        <LoadingIndicator message="Loading the form..." />
+      )}
```

Comment:
I moved the loading indicator to just wrap around the form so that the heading tag is always on the page and focusable